docs/spec/functional_spec.rst

```diff
@@ -244,46 +259,146 @@ Liquidation Auctions
+| kit           | nat                   | The amount of kit to be bid                                             |
++---------------+-----------------------+-------------------------------------------------------------------------+
+
+
 Claim the collateral from a winning auction bid
 -----------------------------------------------
 
-``liquidation_auction_claim_win: int``
+Claim the rewards of a completed liquidation auction. Fails if the sender is
+not the auction winner, if the auction is still ongoing, or if the completed
+auction still has unprocessed liquidation slices. If the operation is
+successful, a tez payment is made to ``Tezos.sender`` with the auction
+winnings.
+
+``liquidation_auction_claim_win: nat``
+
++---------------+-----------------------+-------------------------------------------------------------------------+
+| Parameter     |      Field Type       | Description                                                             |
++===============+=======================+=========================================================================+
+| auction_id    | nat                   | The unique identifier of the completed auction                          |
++---------------+-----------------------+-------------------------------------------------------------------------+
+
 
 Gather won collateral for a subsequent claim
 --------------------------------------------
 
+Internal. Receive a liquidation slice (tez) from a burrow.
+
 ``receive_slice_from_burrow: (pair address nat)``
 
++---------------+-----------------------+-------------------------------------------------------------------------+
+| Parameter     |      Field Type       | Description                                                             |
++===============+=======================+=========================================================================+
+| owner         | address               | The burrow owner's address                                              |
++---------------+-----------------------+-------------------------------------------------------------------------+
+| id            | nat                   | The caller's ID for the burrow sending the slice                        |
++---------------+-----------------------+-------------------------------------------------------------------------+
+
+
 Maintenance entrypoints
 =======================
 
 Perform Checker internal maintenance
 ------------------------------------
 
+Perform housekeeping tasks on the contract state. This includes:
+
+#. updating the system parameters;
+#. accruing burrowing fees to the cfmm;
+#. updating auction-related info (completing an old / starting a new auction);
+#. processing a limited number of liquidation slices from completed auctions;
+#. updating the index by consulting the oracle.
+
+This operation credits an amount of kit (that is a function of time passed
+since the last time ``touch`` was called) to ``Tezos.sender``.
```

Comment:
To me, it would be clearer to have the word "reward" somewhere in the description, maybe something like:

```suggestion
since the last time ``touch`` was called) as a reward to ``Tezos.sender``.
```